### PR TITLE
docs(CONTRIBUTING): prepend `https://` to link

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
fixes a link thats going to a folder (or a branch) that doesn't exist anymore

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ changes to this document in a pull request.
 
 ## Recommendations
 
-- Create a [topic branch](git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch) on your fork for your
+- Create a [topic branch](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows) on your fork for your
```
Comment (uncenter) [inline CONTRIBUTING.md:15]:
```suggestion
- Create a [topic branch](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch) on your fork for your
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Review by sgoudham — APPROVED** [review]

**Comment by somerand0mcat** [comment]
hey @sgoudham would you mind telling me some feedback on how to improve my [recent userstyles PRs](https://github.com/catppuccin/userstyles/pull/1183) and make it better?

**Comment by uncenter** [comment]
> @sgoudham btw would you mind telling me some feedback on how to improve my [recent userstyles PRs](https://github.com/catppuccin/userstyles/pull/1183) and make it better?

Hammy is on the core team, and we on the userstyles staff team have given you plenty of feedback. I'm not sure what else Hammy could offer you here - and this isn't even a userstyles PR! 😅

**Comment by somerand0mcat** [comment]
> Hammy is on the core team, and we on the userstyles staff team have given you plenty of feedback. I'm not sure what else Hammy could offer you here - and this isn't even a userstyles PR! 😅

i mean.. why not

**Comment by somerand0mcat** [comment]
> Hammy is on the core team, and we on the userstyles staff team have given you plenty of feedback. I'm not sure what else Hammy could offer you here - and this isn't even a userstyles PR! 😅

i mean.. why not

**Comment by uncenter** [comment]
> > Hammy is on the core team, and we on the userstyles staff team have given you plenty of feedback. I'm not sure what else Hammy could offer you here - and this isn't even a userstyles PR! 😅
> 
> i mean.. why not

Since you are needlessly pinging a core team member for feedback on a userstyles issue. And since you have been struggling to integrate our comments over your PRs, I'm not sure "more feedback" is helpful for you or worth the time by Hammy.

**Comment by somerand0mcat** [comment]
> Since you are needlessly pinging a core team member for feedback on a userstyles issue. Not to mention you have been struggling to integrate our comments over your PRs- I'm not sure "more feedback" is helpful for you or worth the time to give from Hammy.

maybe i'll just stop asking hammy for feedback ig

> struggling to integrate our comments over your PRs

comments like what, better classnames?

**Comment by uncenter** [comment]
> maybe i'll just stop asking hammy for feedback ig

Yes, this is what I said. Another suggestion is you don't need to restate others's feedback every time it is given, just thumbs up and move on.

> comments like what, better classnames?

Again, I've given you plenty of feedback recently (yes, better classnames!) and *this is not a userstyles PR*. Willing to discuss this further but I'm not sure what you are looking for? We had this exact same discussion less than 48 hours ago on a related PR.

**Pull request merged**